Documentation/install/upgrade.rst

```diff
@@ -322,6 +322,12 @@ New ConfigMap Options
     default ConfigMap. Existing deployments will continue to use ``kvstore``
     unless opted into new behavior via the ConfigMap.
 
+  * ``enable-non-local-node-identity`` has been added to enable a new identity
+    for non-local cluster nodes. This allows to treat local and remote cluster
+    nodes differently from a policy perspective. The option is enabled by
+    default for new deployments when generated via Helm and disabled by default
+    for existing deployments.
```

Comment:
What does it mean that it is "disabled by default for existing deployments" ? The next minor release X.Y upgrade will cause the new default to be picked up if users follow the [current upgrade guide](http://docs.cilium.io/en/latest/install/upgrade/#step-2-option-a-generate-yaml-using-helm-recommended)

Reply:
Ah, I guess that rather than following the (2A) options, the user must follow the (2B) options.